### ISSUE TITLE
driver-only: ECP.PSA starter

### ISSUE THREAD

**Issue by mpg** [Mbed-TLS/mbedtls#7404]
**Context:** see #6839.
**Depends on:** #7390

- [ ] Add an `all.sh` component like the current `accel_all_ec_algs_use_psa` but with `ECP_C` and `ECP_LIGHT` disabled¹, starting from `crypto_full` config with all of PK (inc. parse and write) disabled (as well as RSA which would re-enable it). 
- [ ] Fix the failures² - in my test run there were 7 unique sources of failure:
```
    271   !"Sanity check not implemented for this key type" at line 800, src/psa_exercise_key.c
      7   status == expected_status at line 9214, suites/test_suite_psa_crypto.function
      5   (psa_key_derivation_output_key(&derived_attributes, &operation, &derived_key)) == PSA_SUCCESS at line 9152, suites/test_suite_psa_crypto.function
      2   (psa_import_key(&attributes, key_data->x, key_data->len, &key)) == PSA_SUCCESS at line 2711, suites/test_suite_psa_crypto_driver_wrappers.function
      2   (psa_import_key(&attributes, key_data->x, key_data->len, &key)) == PSA_SUCCESS at line 2641, suites/test_suite_psa_crypto_driver_wrappers.function
      2   (psa_import_key(&attributes, key_data->x, key_data->len, &key)) == PSA_SUCCESS at line 2526, suites/test_suite_psa_crypto_driver_wrappers.function
      1   (psa_key_derivation_output_key(&attributes, &operation, &derived_key)) == PSA_SUCCESS at line 8998, suites/test_suite_psa_crypto.function
```
- [ ] Add a reference component and add the pair to `analyze_outcomes.py` - then either reach parity if easy enough (and the PR is not already large), or create follow-up issue about it.

¹ Note: `ECP_C` can be directly disabled, but for `ECP_LIGHT` since it's auto-enabled in `build_info.h` the way to disable it is to make sure we don't enable anything that would auto-enable it. To be sure we can do the usual grepping the object file: `not grep mbedtls_ecp library/ecp.o`.)

² Note: unless each fix is small and simple, it might be desirable to do this in multiple PRs, if that can be done. (For example, if it's possible to fix the source of the failure before adding the component that will make them fail, a bit like we did 7387 to 7389 pre-emptively before 7390.)

**Comment by gilles-peskine-arm** [comment]
> in my test run there were 7 unique sources of failure

The person who codes this up will of course need to test locally, but if there are any design discussions, it would help to know which commit you tested with, to match line numbers.

**Comment by mpg** [comment]
Good point. I tested with a local commit on top of the ECP-light prototype branch, as all changes related to ECP-light have not been merged yet. Here's [the branch](https://github.com/mpg/mbedtls/tree/investigate-no-ecp) and the [commit](https://github.com/Mbed-TLS/mbedtls/commit/23e6a3a1d21a3ccc479c6bcb5cf4398efe449b2a). Note: as this was just a local test, I modified the existing `accel_all_ec_algs_use_psa()` component but as the task description says, we should actually create a new one.

I've updated the description to include the text of the failing assertion. I generated this with:
```sh
tests/scripts/all.sh -k test_psa_crypto_config_accel_all_ec_algs_use_psa | tee all.log
grep -B1 'at line' all.log | sed -n '/"$\|==/ N;s/\n  / /p' | sort | uniq -c | sort -rn
```
The same can be done by the person doing the work (likely @valeriosetti), just replacing `test_psa_crypto_config_accel_all_ec_algs_use_psa` with the new component added in the first step.
